README.md

```diff
@@ -84,9 +84,30 @@ t({
 })
 ```
 
-
 When new texts are created or existing texts are modified in the application please leave a message in the OlympusDao app-translation channel for the translators to translate them.
 
+### Resolving merge conflicts with translations
+
+```bash
+$ git diff
+# shows two commits in conflict below (fbdd867,e6e0919)
+diff --cc src/locales/translations
+index fbdd867,e6e0919..0000000
+--- a/src/locales/translations
++++ b/src/locales/translations
+
+cd src/locales/translations
+# first commit
+git checkout fbdd867
+# merge in second commit
+git merge e6e0919
+git commit
+
+cd ../../..
+git add src/locales/translations
+git commit
+```
+
```

Comment:
added because there was originally a merge conflict, which took some googling to resolve ^^^